.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,60 @@
+# Update robottelo image on quay.
+name: update_robottelo_image
+
+on:
+  push:
+    branches:
+      - master
+      - 6.*.z
+  repository_dispatch:
+    types:
+      - airgun-pr-merged
+      - nailgun-pr-merged
+
+jobs:
+  robottelo_container:
+    name: Update Robottelo container image on Quay.
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get branch info
+        id: repo_branch
+        run: |
+          BRANCH="${{ github.event.client_payload.branch }}"
```

Comment:
I would rather use the below approach so that evaluating the ${{ github.event.client_payload.branch }} would not be a problem. Did you test this for both even types?  This comment is non-blocking 
```

    steps:
      - name: Check event type
        if: github.event_name == 'push'
        run: echo "This workflow was triggered by a push event"
      - name: Check event type
        if: github.event_name == 'repository_dispatch'
        run: echo "This workflow was triggered by a repository dispatch event"
```